README.md

```diff
@@ -1,9 +1,17 @@
+> This is a **Sanity Studio v3** plugin.
+
 # Sanity Workflow Demo Plugin Example
 
 With Sanity Studio you can [customize your content tools to support arbitrary workflows like assignment and content pipelines](https://www.sanity.io/docs/custom-workflows).
 
 This plugin is distributed as a **reference implementation** of these customization APIs and is not considered to be a feature-complete implementation of what workflow management requires in production. It is a starting point intended to be forked and customized to the needs of your organization and content creators.
```

Comment:
```suggestion
This plugin is distributed as an **example implementation** of customization APIs in the Sanity Studio V3 and is not considered to be a feature-complete implementation of what workflow management requires in production. It is meant as a starting point intended to be forked and customized to the needs of your organization and content creators, or simply as an illustration of what is possible in Sanity Studio V3.
```

Lets not call this a reference implementation. There are some patterns here I wish we could have avoided, like the WorkflowSignal component feeding into the context via top-level component rendering in the document form.

Reply:
Good call. I'd rather not have that either but there's presently no other way to feed a single async piece of data from a document up to a context that multiple badges and actions can read from.